pkg/testcase/helm.go

```diff
@@ -88,3 +111,29 @@ func TestDeployRancher(helmVersion, imageVersion string) {
 	}
 	fmt.Println("\nRancher URL:", rancherUrl)
 }
+
+func helmArgsBuilder(flags *customflag.FlagConfig) (finalArgs string) {
+	helmArgs := flags.ExternalFlag.HelmChartsFlag.Args
+	if strings.Contains(helmArgs, ",") {
+		argsSlice := strings.Split(helmArgs, ",")
+		for _, arg := range argsSlice {
+			if !strings.Contains(finalArgs, arg) {
+				finalArgs += fmt.Sprintf("--set %s ", arg)
+			}
+		}
+	} else {
+		finalArgs = fmt.Sprintf("--set %s ", helmArgs)
+	}
+
+	if strings.Contains(flags.ExternalFlag.RancherVersion, "v2.7.12") {
```

Comment:
Is this only valid for this one version? Or should we remove this and expect any `extraEnv` stuff to be added into the helmChartArgs themselves when relevant?

Reply:
I think so as I have not seen those args being used / needed in v2.8+

Reply:
But if needed, we can remove this check and let user define those args for v2.7.x

Reply:
yeah let's remove it and document in testing.md! I think it'll be relevant in other versions later on so I'd rather be future-proof here

Reply:
Updated